src/command_modules/azure-cli-network/azure/cli/command_modules/network/_format.py

```diff
@@ -85,18 +85,24 @@ def transform_vpn_connection_list(result):
 
 def transform_vpn_connection(result):
 
-    properties_to_strip = \
-        ['virtual_network_gateway1', 'virtual_network_gateway2', 'local_network_gateway2', 'peer']
-    for prop in properties_to_strip:
-        prop_val = getattr(result, prop, None)
-        if not prop_val:
-            delattr(result, prop)
-        else:
-            null_props = [key for key in prop_val.__dict__ if not prop_val.__dict__[key]]
-            for prop in null_props:
-                delattr(prop_val, prop)
+    if result:
```

Comment:
You are correct--it was needed for another issue I discovered whilst fixing these.